src/views/tutorials-landing/components/cross-product-section/cross-product-section.module.css

```diff
@@ -0,0 +1,41 @@
+.root {
+	background-color: var(--token-color-surface-primary);
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	border-radius: 15px;
+	display: flex;
+	flex-direction: column;
+	row-gap: 32px;
+
+	/* TODO rexport SVGs from Figma during vqa/polish */
+
+	@nest html[data-theme='light'] & {
+		background-image: url('img/background-light.svg');
+	}
+
+	@nest html[data-theme='dark'] & {
+		background-image: url('img/background-dark.svg');
+	}
```

Comment:
What is `@nest`?  I've never seen this selector in practice.

Reply:
We use it a bit ito customize properties based on which theme is actively selected:
https://github.com/search?q=repo%3Ahashicorp%2Fdev-portal%20%40nest&type=code